day01/index.js

```diff
@@ -0,0 +1,43 @@
+const R = require('ramda');
+const fs = require('fs');
+const path = require('path');
+
+const inputData = fs.readFileSync(path.resolve(__dirname, 'data.txt'), 'utf8');
+
+const convertInputToArr = input =>
+  input
+    .toString()
+    .split(/\n|\$/)
+    .map(item => parseInt(item, 10));
+
+const getFuelRequirementByMass = mass => mass.map(m => Math.floor(m / 3) - 2);
+
+const reduceFuelArr = fuelArr =>
+  fuelArr.reduce((total, currentValue) => total + currentValue);
+
+const getFuelForMass = R.pipe(
+  convertInputToArr,
+  getFuelRequirementByMass,
+  reduceFuelArr
+);
```

Comment:
Love this, @audifred! More, more, more! 😄

Reply:
👍